packages/tools/generator-component/generators/app/templates/__package__.json

```diff
@@ -6,9 +6,9 @@
   "module": "dist/f-<%= name.default %>.es.js"<% } %><% if(config.needsBundlewatch) { %>,
   "maxBundleSize": "<%= bundlewatchMaxSize %>kB"<% } %>,
   "files": [
-    "dist"<% if(config.needsComponentTests) { %>,
-    "test-utils"<% } %><% if(config.needsTestingApiMocks) { %>,
-    "src/services"<% } %>
+    "dist",
+    "test-utils",
+    "src/services"
```

Comment:
I don't think this needs to be exported, does it?

Reply:
I agree – I think this only needs to be exported in certain circumstances?

Reply:
ah yep you're right the `src/services` doesn't - will revert that now. The `test-utils` would be needed even if there were no component tests as its used for our accessibility tests in CoreWeb - hence why I removed that conditional :)

Reply:
> Shall we create desktop/mobile visual tests as part of the generator?

Added the starter template for these 😄  https://github.com/justeat/fozzie-components/pull/1525/files#diff-34c172c4d98130674099f6c035932983bb7c21369c31f26325836091386a6de8